Pages/prod-listing/prod-list.css

```diff
@@ -0,0 +1,330 @@
+@import url('https://fonts.googleapis.com/css2?family=Caveat:wght@500&family=Comforter&family=Nunito:wght@400;700;800&family=Roboto+Condensed:wght@300;400;700&family=Roboto:wght@700;900&family=Shizuru&family=Smooch&display=swap');
+
+:root {
+    --color-primary: #5ec576;
+    --color-secondary: #ffcb03;
+    --color-tertiary: #ff585f;
+    --color-primary-darker: #4bbb7d;
+    --color-secondary-darker: #ffbb00;
+    --color-tertiary-darker: #fd424b;
+    --color-primary-opacity: #5ec5763a;
+    --color-secondary-opacity: #ffcd0331;
+    --color-tertiary-opacity: #ff58602d;
+    --gradient-primary: linear-gradient(to top left, #39b385, #9be15d);
+    --gradient-secondary: linear-gradient(to top left, #ffb003, #ffcb03);
+    --color-white:#f3f3f3;
+    --background-white:rgba(255,255,255,0.95);
+    --light-grey:#333;
+    --lighter-grey:#222;
+    --grey-color:#808080;
+    --color-beige:#F5F5DC;
+
+  }
+
+
+*{
+    margin: 0;
+    box-sizing: border-box;
+}
+
+
+html{
+    scroll-behavior: smooth;
+}
+
+body{
+    background-color: var(--color-white);
+    overflow-y: scroll;
+
+}
+
+.main-navbar{
+    display:flex;
+    background-color: var(--background-white);
+    height:12rem;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+    position:fixed;
+    top:0;
+    left:0;
+    right:0;
+    z-index: 11;
```

Comment:
Just a suggestion, if you want can do according to the images.
![Screenshot 2022-02-13 at 9 25 52 AM](https://user-images.githubusercontent.com/79623546/153738148-9f65c9b7-edf1-4a25-b6d6-2ce8bc1d384f.png)
![Screenshot 2022-02-13 at 9 28 21 AM](https://user-images.githubusercontent.com/79623546/153738154-8400b5eb-b819-4e22-a061-bbbca0ae1320.png)
![Screenshot 2022-02-13 at 9 29 30 AM](https://user-images.githubusercontent.com/79623546/153738163-ed3bef23-426c-4377-867a-0d2d1c530912.png)